sourcery/sourcery.rb

```diff
@@ -0,0 +1,62 @@
+module Fastlane
+  module Actions
+    class SourceryAction < Action
+      def self.run(params)
+        action = params[:action]
+
+        case action
+        when "install"
+            require 'brew'
+            brew(command: 'install sourcery')
+        when "run"
+            sources = params[:sources]
```

Comment:
All this params have to be specified in `FastlaneCore::ConfigItem`.

Reply:
@AndriiMoskvin Done